prow/tide/tide.go

```diff
@@ -1146,6 +1147,17 @@ func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
 			continue
 		}
 
+		// Ensure tide context if enabled
+		if tideConfig.TideGithubContext != nil && *tideConfig.TideGithubContext {
+			err = c.ensureTideContext(pr, log)
```

Comment:
Please add tests for this

Reply:
Added some, let me know if it makes sense, followed examples from other tests on using the fgc